ansible/playbooks/azure.yml

```diff
@@ -18,7 +18,7 @@
     - name: Install azure-cli
       ansible.builtin.shell:
         cmd: >
-          sudo ./tmp/azure_cli.sh
+          sudo /tmp/azure_cli.sh
```

Comment:
why can't we just use apt-get install -y azure-cli ? even the script itself does not like it:

```
# Do not be in the habit of executing scripts from the internet with root-level access to your machine. Only trust    #
# well-known publishers.                                                                                              #
```

Reply:
There's two options there. Running the script is the "easiest" option 1 but option 2 to setup explicitly is also there. Having IaC steps that hides away what we're configuring in an external script is not a good option.